pkg/api/api.go

```diff
@@ -183,29 +178,41 @@ func (a *API) getHeader(w http.ResponseWriter, r *http.Request) (int, error) {
 			"slot":     slot,
 			"proposer": proposer,
 		}).WithError(err).Debug("failed getHeader")
-		return http.StatusBadRequest, err
+
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
can we extract this code into a function? In order to avoid code duplication

Reply:
If you like. I'm not sure it's needed in this particular case, but If you prefer - I'm fine.